src/egon/data/datasets/heat_demand/__init__.py

```diff
@@ -38,9 +38,36 @@
 from sqlalchemy import Column, String, Float, Integer, Sequence, ForeignKey
 from sqlalchemy.ext.declarative import declarative_base
 
-Base = declarative_base()
+from egon.data.datasets import Dataset
+
+# class for airflow task management (and version control)
+class HeatDemandImport(Dataset):
+
+    # data_config = egon.data.config.datasets()
```

Comment:
The commented out parts of the code confuse me a bit , are they still needed?

Reply:
These lines were added by @EvaWie . I'm not sure why she added them, maybe because the sources and targets will be part of the dataset in the future. 
But since we don't know how this will be implemented, we could delete them from my point of view.